sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioUtils.scala

```diff
@@ -154,8 +156,15 @@ object AlluxioUtils extends Logging {
       alluxioCmd = conf.getAlluxioCmd
       checkAlluxioNotSupported(conf)
 
-      if (!isInit) {
-        if (conf.getAlluxioAutoMountEnabled) {
+      if (isConfiguredReplacementMap(conf)) {
+        // replace-map is enabled, if set this will invalid the auto-mount
+        if (!isInitReplaceMap) {
+          alluxioPathsToReplaceMap = getReplacementMapOption(conf)
+          isInitReplaceMap = true
+        }
+      } else if (conf.getAlluxioAutoMountEnabled) {
+        // auto-mound is enabled
+        if (!isInitMountPointsForAutoMount) {
```

Comment:
similar comment here, since we do initialization once we can't change the bucket regex at run time and have it work.  This is fine just might want to comment on it.   @GaryShen2008